src/libmongoc/tests/test-mongoc-sdam-monitoring.c

```diff
@@ -827,6 +829,7 @@ _test_heartbeat_fails_dns (bool pooled)
    size_t i;
 
    context_init (&context);
+   start = bson_get_monotonic_time ();
```

Comment:
Earlier `_test_heartbeat_events` had a comment that the start time should be taken before `mongoc_client_pool_pop`. Should there be a similar comment here?